src/Extension/FluentExtension.php

```diff
@@ -161,6 +151,44 @@ public function getLocalisedFields($class = null)
     }
 
     /**
+     * @param string $field
+     * @param string $type
+     * @param string|null $class
+     * @return bool
+     */
+    public function isFieldLocalised($field, $type, $class = null)
```

Comment:
We probably should move the `$filter = Config::inst()->get($class, 'translate', Config::UNINHERITED);` check into here too.